temporal-sdk/src/main/java/io/temporal/activity/Activity.java

```diff
@@ -26,230 +29,40 @@
 import io.temporal.internal.sync.WorkflowInternal;
 
 /**
- * An activity is the implementation of a particular task in the business logic.
- *
- * <h2>Activity Interface</h2>
- *
- * <p>Activities are defined as methods of a plain Java interface annotated with {@literal @}{@link
- * ActivityInterface}. Each method defines a single activity type. A single workflow can use more
- * than one activity interface and call more than one activity method from the same interface. The
- * only requirement is that activity method arguments and return values are serializable to a byte
- * array using the provided {@link io.temporal.common.converter.DataConverter} implementation. The
- * default implementation uses JSON serializer, but an alternative implementation can be configured
- * through {@link io.temporal.client.WorkflowClientOptions.Builder#setDataConverter(DataConverter)}.
- *
- * <p>Example of an interface that defines four activities:
- *
- * <pre><code>
- * {@literal @}ActivityInterface
- * public interface FileProcessingActivities {
- *
- *     void upload(String bucketName, String localName, String targetName);
```

Comment:
Where'd all this move to? The conceptual docs? I would still want a link to that in these docstrings at minimum. Being able to get to there from here is important. I'm not seeing where it went in the linked example page.

Reply:
This content was moved to https://docs.temporal.io/docs/java-activities
The SDK README will contain an explanation of where to find additional conceptual and implementation information. I am against putting links directly in the comment/doc strings for maintainability reasons.

Reply:
Yeah, I can get that. We'll want some kind of solution though -- for example it's standard practice to have examples in docstrings in Rust code and they actually get compiled and run as tests to ensure they are always up to date.

So, maybe that means extracting the docs page from these comments if the goal is to avoid maintaining links, but it would be ideal to still have code examples easily accessible from the code. 

Whatever tool is using these annotations, can we just use it to insert backlinks?